### ISSUE TITLE
Move over to argparse

### ISSUE THREAD

**Issue by IceflowRE** [scour-project/scour#278]
Currently optparse is used and deprecated and stopped developing since 2011.
Also the options should be moved to a `dict`, it is a hassle to work with `optparse.Values` in third-party software.

It was already a bit discussed in #112 .

**Comment by Ede123** [comment]
Indeed, this has been a "low priority" improvement for the last five years, see https://github.com/scour-project/scour/pull/44. ;-)

PRs welcome.

For now you can make use of `sanitizeOptions` (which was introduced in the linked PR) which lets you make use of "random" objects (so you don't absolutely need to handle `optparse.Values`).
